java/src/com/liferay/nativity/control/win/WindowsNativityUtil.java

```diff
@@ -65,15 +65,17 @@ private static void _load() {
 
 			_loaded = true;
 
+		    System.out.println("Successfully loaded " + nativityDllName);
```

Comment:
This println is super useful though. There is a lot of work involved to make this right